### ISSUE TITLE
An Mm3d model pack would be a useful thing

### ISSUE THREAD

**Issue by mick-p1982** [zturtleman/mm3d#48]
http://www.misfitcode.com/misfitmodel3d/models.html

> I do not currently have any sample 3D models available for download on this site. You can find some free 3D models at the following web sites. If you want to load these models in Misfit Model 3D you should check the Features page to see which model formats are supported by the current version. 

Do we have test models that demonstrate the package's features? These are important resources for developers to have access to.

**Comment by mick-p1982** [comment]
Can anyone share some animated models (both kinds ideally) that are MM3D or that will import?

Also, is this (https://clover.moe/wp-content/uploads/2018/09/mm3d-logo-remake.png) model available?

**Comment by mick-p1982** [comment]
I think I've found an animated model in: tests/libmm3d/data/model_equal_test.mm3d. It looks like it has both kinds in the form of identical animations.

**Comment by mick-p1982** [comment]
https://github.com/zturtleman/mm3d/blob/8c6c6ea29e2ad423d35ed1c2147f09c68f49d94b/src/implui/animwidget.cc#L545

I'm not sure what this code is supposed to do. It speeds up the frame animations in the model_equal_test.mm3d. 

When I rewrote it I assumed it was setting a maximum frame rate, but it's actually a minimum as written. I changed it to 60fps thinking that a better maximum than 20. But it's actually the other way around.

**Comment by zturtleman** [comment]
It probably only affects how often model view port is redrawn, not playable speed, so that less than 20 frame per-second animations can interpolate between frames. (I haven't confirmed if it affects playback speed or not.) Seems like it should always update at monitor refresh rate or be a setting in the missing preferences window.

**Comment by mick-p1982** [comment]
You're right, I hadn't thought about it that way. I still can't grasp what the rationale for it is... without comments accompanying the code.

It only fell onto my radar because I misinterpreted it and the repercussions were much less subtle in that case. I think it needs a case to be made for it, since is subtly changes playback. I think using timers to render animations is probably flawed... but it may be the only way to do low-frame-rate playback as-if the display itself it limited to that frame-rate. I think probably the frames should just be interpolated and played back at SwapBuffer speed.
